re 351 retain the link to the macro concept of append only log (#353)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

The term "claim" is defined in {{RFC8392}}.

Statement Sequence:

A sequence of Signed Statements captured by a Verifiable Data Structure.

: see Verifiable Data Structure

Append-only Log:

A Statement Sequence comprising the entire registration history of the Transparency Service.

> To make append-only verifiable and transparent, the Transparency Service defines how Signed Statements are made available to Auditors.

Reviewer: ```suggestion
To make the append-only property verifiable, the Transparency Service defines how Signed Statements are made available to Auditors.
```

I am not sure how to parse the original sentence, two things that are easier for me to understand are "append-only is a property of a sequence, we

> To make append-only verifiable and transparent, the Transparency Service defines how Signed Statements are made available to Auditors.

Reviewer: ```suggestion
To make the append-only property verifiable and transparent, the Transparency Service defines how Signed Statements are made available to Auditors.
```

Feels a bit clearer to me, but happy to follow native speaker advice here :)

Author: Ah, yes your change is better

> To make the append-only property verifiable and transparent, the Transparency Service defines how Signed Statements are made available to Auditors.

Reviewer 2: ```suggestion
To make the Append-only property verifiable and transparent, the Transparency Service defines how Signed Statements are made available to Auditors.
```

Append-Only:

> : A property required for a verifiable data structure to be applicable to SCITT, ensuring that the Statement Sequence cannot modified, deleted or reordered.

Reviewer 2: ```suggestion
: a property required for a verifiable data structure to be applicable to SCITT, ensuring that the Statement Sequence cannot be modified, deleted or reordered.
```

> : once committed to the verifiable data structure, a Signed Statement cannot be modified, deleted, or reordered; hence its Receipt provides an offline verifiable proof of Registration.

Reviewer 3: ```suggestion
: A property required for a verifiable data structure to be applicable to SCITT, ensuring that the Statement Sequence cannot modified, deleted or reordered.
```

Reviewer 3: consider moving this definition above "Append-Only Log"

Reviewer 2: Note: I've tweaked a bit to add "...be modified..."

A Transparency Service MAY accept a Signed Statement with content in its unprotected header, and MAY use values from that unprotected header during verification and registration policy evaluation.

> However, the unprotected header of all Signed Statements committed to the Verifiable Data Structure MUST be empty.

Reviewer 3: ```suggestion
However, the unprotected header of a Signed Statement MUST be set to the empty map, before the signed statement can be included in a Statement Sequence.
```

> A Transparency Service MUST replace the unprotected header with an empty unprotected header before committing to the Verifiable Data Structure.

Reviewer 3: ```suggestion
```

Reviewer 3: Duplicate?

Author: ah yeah, but also, I was replacing "included" with committed elsewhere. In the context of a Statement Sequence though, I am fine with included

In some deployments a special role such as an Auditor might require and be given access to both the Transparency Service and related Adjacent Services.

> Transparency Services' Verifiable Data Structures MAY carry only cryptographic metadata (e.g. a hash), rather than the complete Signed Statement, which does not raise immediate privacy concerns.

Reviewer 3: ```suggestion
Transparency Services' can leverage Verifiable Data Structures which only retain cryptographic metadata (e.g. a hash), rather than the complete Signed Statement, as part of a defense in depth approach to maintaining confidentiality.
```

Reviewer 3: No reason for MAY, but I'm not attached to this change

Reviewer: @OR13 "Transparency Services" then?

Author: happy for that to be committed

In contrast, Transparency Services can be held accountable and blamed by an Auditor that replays the Sequence of Signed Statements captured in their Verifiable Data Structure to confirm that a contested Receipt is valid and was correctly registered.

Transparency Services can provide consistency proofs allowing Auditors to check if a set of Receipts were issued from a single Verifiable Data Structure, without replaying individual Signed Statements.

> Transparency Services can provide proof allowing Auditors to check that a transparent Statement Sequence is complete and consistent and contains no ommisions.

Reviewer 3: ```suggestion
Certain Verifiable Data Structures enable a Transparency Service to prove properties of it's Statement Sequence.
For example, proving a specific signed statement is included in the sequence, or that the sequence has only been extended (append only property) since the last time such a proof was created.
```

Author: I am fine with, and like, this change
